src/index.js

```diff
@@ -105,23 +130,25 @@ export default {
             addWorkspaceOpenedWatch: addWorkspaceOpenedWatch,
             removeWorkspaceOpenedWatch: removeWorkspaceOpenedWatch,
         }*/
-    },
-    onunload: () => {

```

Comment:
this is just me moving the `onunload` from being its own function to being the return value of `onload`. This should make unloading simpler - for example, `sidebar.js` now returns its unload function directly so that the `onunload` has access to it